src/main.scss

```diff
@@ -1,3 +1,91 @@
+// Breakpoints
+$large: 80rem;
+
+// Palette
+$root: #D9A521;
+$bright: #D9CB2B;
+$deep: #D97416;
+$complement: #420BD9;
+// $countercomplement: #36B7D9;
```

Comment:
Left this (and the secondary color) since they round out the colour palette, but we don't need them thus far.